src/eherkenning/mock/backends.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import re
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import gettext_lazy as _
+
+from digid_eherkenning.backends import BaseBackend
+from digid_eherkenning.utils import get_client_ip
+
+logger = logging.getLogger(__name__)
+
+UserModel = get_user_model()
+
+
+class eHerkenningBackend(BaseBackend):
+    service_name = "eHerkenning"
+    error_messages = dict(
+        BaseBackend.error_messages,
+        **{
+            "eherkenning_no_kvk": _(
+                "Login failed due to no KvK being returned by eHerkenning."
+            ),
+            "eherkenning_len_kvk": _(
+                "Login failed due to no KvK having more then 8 digits."
+            ),
+            "eherkenning_num_kvk": _("Login failed due to no KvK not being numerical."),
+        }
+    )
```

Comment:
Messages for logging shouldn't be translated. There's no use case, and it's confusing when they end up in log aggregating systems translated into the default language of the project. I know the `digid-eherkenning` library does this too, but that's a mistake and I filed an issue about it ([link](https://github.com/maykinmedia/django-digid-eherkenning/issues/49)).